test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -397,6 +397,21 @@ class test_uct_cm_sockaddr : public uct_test {
         m_listen_addr  = GetParam()->listen_sock_addr;
         m_connect_addr = GetParam()->connect_sock_addr;
 
+        const ucs::sock_addr_storage &src_sock_addr =
+                                                GetParam()->source_sock_addr;
+        if (src_sock_addr.get_sock_addr_ptr() != NULL) {
+            char src_addr_cstr[UCS_SOCKADDR_STRING_LEN];
+            int sa_family   = src_sock_addr.get_sock_addr_ptr()->sa_family;
+            const char *ret = inet_ntop(sa_family,
+                                        src_sock_addr.get_sock_addr_in_buf(),
+                                        src_addr_cstr, UCS_SOCKADDR_STRING_LEN);
+            EXPECT_EQ(src_addr_cstr, ret);
+            UCS_TEST_MESSAGE << "testing with RDMA_CM_SOURCE_ADDRESS="
+                             << src_addr_cstr;
```

Comment:
can we add this to line 437?

Reply:
IMO better to log near action

Reply:
but then log appears twice per test

Reply:
what do you mean?
i see only 1:
```
[ RUN      ] sockaddr/test_uct_cm_sockaddr.cm_server_reject/34 <sockaddr/ib3>
[     INFO ] Testing component: tcp
[     INFO ] testing with RDMA_CM_SOURCE_ADDRESS=172.24.1.10
[     INFO ] Testing 172.24.1.10:41656 Interface: ib3
[       OK ] sockaddr/test_uct_cm_sockaddr.cm_server_reject/34 (1 ms)
```

Reply:
1. [     INFO ] Testing component: tcp
2. [     INFO ] testing with RDMA_CM_SOURCE_ADDRESS=172.24.1.10
3. [     INFO ] Testing 172.24.1.10:41656 Interface: ib3

Reply:
should be:
[     INFO ] Testing tcp on 172.24.1.10:41656 interface: ib3 with RDMA_CM_SOURCE_ADDRESS=172.24.1.10

Reply:
i will complicate the code in case if config is not set, are you fine with this?

Reply:
let's try to make simple change to have less log messages, if not possible - waive